easy_thumbnails/conf.py

```diff
@@ -96,27 +95,26 @@ class Settings(AppSettings):
     thumbnail when using the :ref:`thumbnail_tag` rather than failing silently.
     """
 
-    THUMBNAIL_DEFAULT_STORAGE = (
-        'easy_thumbnails.storage.ThumbnailFileSystemStorage')
+    THUMBNAIL_DEFAULT_STORAGE = "easy_thumbnails.storage.ThumbnailFileSystemStorage"
     """
     The default Django storage for *saving* generated thumbnails.
     """
-    THUMBNAIL_MEDIA_ROOT = ''
```

Comment:
For consistency, what's your logic about single and double quotes? I prefer single quotes for strings used by the program, for instance keys in dicts or to access object members, but double quotes for everything readable by humans.

Reply:
Personally, I always prefer single quotes. However, in my opinion, the easiest way is to put up with black and use doubles everywhere.
Black did not replace the single quotation marks in the commented code as far as I can see.

Reply:
I used to care about quotes, but the lighter load of not worrying and being consistent with a standard has been refreshing (we black all our code at work now).